openapi-cli/src/test/java/io/ballerina/openapi/cmd/OpenApiGenServiceCmdTest.java

```diff
@@ -117,18 +117,47 @@ public void testOneOfSchemaGen() throws IOException {
         }
     }
 
+    @Test(description = "Test open-api genservice for successful service generation with OneOf schema type")
```

Comment:
Shouldn't this be
```suggestion
    @Test(description = "Test OpenAPI genservice for successful service generation with OneOf schema type having arrays")
```
?

Reply:
+1